jnigen/lib/src/bindings/common.dart

```diff
@@ -90,44 +110,154 @@ abstract class BindingsGenerator {
     return args.join(', ');
   }
 
+  String _dartTypeClassName(String className) {
+    return '$typeClassPrefix$className$typeClassSuffix';
+  }
+
+  String dartTypeParams(List<TypeParam> typeParams,
+      {required bool includeExtends}) {
+    if (typeParams.isEmpty) return '';
+    // TODO(HosseinYousefi): resolve the actual type being extended if any
```

Comment:
Don't leave TODOs in the code base without pointing to a GitHub issue. Either fix or file an issue.

Nit: punctuation: "Resolve the actual type being extended, if any."